test/gtest/ucp/test_ucp_am.cc

```diff
@@ -1093,6 +1093,17 @@ class test_ucp_am_nbx_seg_size : public test_ucp_am_nbx_reply {
     {
         modify_config("ADDRESS_VERSION", "v2");
         modify_config("RNDV_THRESH", "inf");
+        if (enable_proto()) {
+            modify_config("PROTO_ENABLE", "y");
+        }
+    }
+
+    static void get_test_variants(std::vector<ucp_test_variant> &variants)
+    {
+        add_variant_values(variants, test_ucp_am_nbx_reply::get_test_variants,
+                           0);
+        add_variant_values(variants, test_ucp_am_nbx_reply::get_test_variants,
+                           1, "proto");
```

Comment:
can we do this in test_ucp_am_nbx_reply (the base class)? or in test_ucp_am_nbx ?